install_pim/docker/installation_docker.rst

```diff
@@ -48,9 +48,11 @@ or from our partner portal if you have access to the enterprise edition. It can
 Using the Docker images
 -----------------------
 
-Every flavor (dev or standard, community or enterprise) comes with a Docker Compose file template ``docker-compose.yml.dist``, ready to be used.
-Copy it as ``docker-compose.yml`` and keep it at the root of your project. You may modify it at your convenience, to change the mapping of the ports
-if you want Apache to be accessible from a port other that 8080, for instance.
+Every flavor (dev or standard, community or enterprise) comes with a Docker Compose file ``docker-compose.yml`` and a ``docker-compose.override.yml.dist``, ready to be used. The ``docker-compose.yml`` file configures all the necessary containers with the default settings to run Akeneo PIM. The ``docker-compose.override.yml.dist`` defines the custom parameters you may change depending on your development environment.
+
+Copy the override file as ``docker-compose.override.yml`` and keep it at the root of your project. You may modify it at your convenience. It defines some ports mapping by default and you can change them if you need, if you want Apache to be accessible from a port other that 8080, for instance.
```

Comment:
Beware, for now there is no `docker-compose.override.yml.dist` in your pim-community-dev PR.